challenges/prototypes.js

```diff
@@ -5,29 +5,40 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
-
+function CuboidMaker(attrs) {
+  this.length = attrs.length;
+  this.width = attrs.width;
+  this.height = attrs.height;
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
-  
+
   Formula for cuboid volume: length * width * height
 */
 
+CuboidMaker.prototype.volume = function() {
+  return `The Volume is: ${ this.length  * this.width * this.height }`;
+}
 
 /* == Step 3: Surface Area Method ==
-  Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
+  Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height.
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
 
+CuboidMaker.prototype.surfaceArea = function() {
+  return `The surface area is: ${ 2 * (this.length * this.width + this.length * this.height + this.width * this.height) }`;
+}
```

Comment:
Used prototypical inheritance to define a function that can be used for new objects created.